src/main/java/seedu/address/model/inventorymodel/InventoryModelManager.java

```diff
@@ -143,6 +139,11 @@ public void updateFilteredItemList(Predicate<Item> predicate) {
         filteredItems.setPredicate(predicate);
     }
 
+    @Override
+    public SortedList<Item> getSortedItemList() {
+        return new SortedList<Item>(filteredItems, ITEM_COMPARATOR);
+    }
```

Comment:
why not return this in `getFilteredItemList()`

Reply:
The method name would be misleading in my opinion

Reply:
@halcon-blanco did that! it turns out this helped iron out bugs so much thanks!